tests/unit/spanner_dbapi/test_cursor.py

```diff
@@ -248,13 +252,13 @@ def test_execute_statement(self):
 
         with mock.patch(
             "google.cloud.spanner_dbapi.parse_utils.classify_statement",
-            return_value=ParsedStatement(StatementType.UPDATE, sql),
+            return_value=ParsedStatement(StatementType.UPDATE, Statement(sql)),
         ):
             cursor.connection._database = mock_db = mock.MagicMock()
             mock_db.run_in_transaction = mock_run_in = mock.MagicMock()
             cursor.execute(sql="sql")
             mock_run_in.assert_called_once_with(
-                cursor._do_execute_update_in_autocommit, "sql WHERE 1=1", None
```

Comment:
Why did this change in this PR?

Reply:
The logic to add WHERE clause has moved to `parse_utils.classify_statement` now which we are mocking here so its same as what is returned as ler line 255